ratis-examples/src/main/resources/log4j.properties

```diff
@@ -16,10 +16,10 @@
 
 
 log4j.rootLogger=INFO, stdout
-log4j.logger.org.apache.ratis.server.impl.RatisServerImpl=DEBUG
-log4j.logger.org.apache.ratis.client.RaftClient=DEBUG
+log4j.logger.org.apache.ratis.server.impl.RatisServerImpl=INFO
+log4j.logger.org.apache.ratis.client.RaftClient=INFO
```

Comment:
Since root level is INFO, I think these two class-specific settings can be removed.

```suggestion
```

Reply:
@adoroszlai Thanks for the suggestion. I will update that and commit again.